core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3040,6 +3048,7 @@ class ReplicaManagerTest {
       transactionalId = transactionalId,
       entriesPerPartition = entriesToAppend,
       responseCallback = appendCallback,
+      apiVersionErrorMapper = genericError
```

Comment:
Is there a reason some of these are generic and some are default? I wonder if we can configure the test to use the "highest" enum

Reply:
I will change all of these to use the `genericError` as the supportedOperation

Reply:
That works for now. Long term, we may want another solution, but we can do in a follow up.